src/composer.json

```diff
@@ -39,7 +39,7 @@
     
     "config": {
         "platform": {
-            "php": "5.3.3"
```

Comment:
Hello Michael, I don't really complain, I just wanted to underline the fact that doing so you loose support for some older versions. Now, it could just be a matter of stating on the project page, which Linux distributions are supported for FOSSology ! And I'd build packages accordingly as well, so would be indeed useful.